src/App.js

```diff
@@ -1,12 +1,86 @@
-import logo from './logo.svg';
 import './App.css';
+import { getDatabase, ref, onValue, push, remove } from 'firebase/database';
```

Comment:
some generic comments here that i dont know where to add! I love the logic of your app! its well formatted and self explanatory